src/cli/cli_tcat.cpp

```diff
@@ -395,35 +398,102 @@ template <> otError Tcat::Process<Cmd("start")>(Arg aArgs[])
  * Done
  * @endcode
  * @par
- * Stops TCAT operation.
+ * Stops TCAT agent and stops TCAT operation.
  * @sa otBleSecureStop
  */
 template <> otError Tcat::Process<Cmd("stop")>(Arg aArgs[])
 {
-    OT_UNUSED_VARIABLE(aArgs);
+    otError error = OT_ERROR_NONE;
 
+    VerifyOrExit(aArgs[0].IsEmpty(), error = OT_ERROR_INVALID_ARGS);
     otBleSecureStop(GetInstancePtr());
 
-    return OT_ERROR_NONE;
+exit:
+    return error;
+}
+
+/**
+ * @cli tcat standby
+ * @code
+ * tcat standby
+ * Done
+ * @endcode
+ * @par
+ * Sets TCAT operation to standby, keeping the agent enabled.
+ * @sa otBleSecureTcatActive
```

Comment:
otBleSecureTcatActive doesn't exist, should it be something else?

Reply:
Indeed, this should be `otBleSecureSetTcatAgentState`. Added this sa link also for the "tcat active" CLI command.